components/Workout/Workout.js

```diff
@@ -54,7 +53,13 @@ export default function Workout({ workout, handleDelete, handleEditWorkout }) {
           />
         </div>
       ) : null}
-      <WorkoutCard>
+      <WorkoutCard
+        $spotlightPadding={
+          spotlightMode
+            ? "padding-top: 0; border-top: 1px solid #0000001a;"
+            : null
+        }
+      >
         {deleteMode ? (
           <ModalOverlay onClick={toggleDeleteMode}>
             <ModalContent onClick={(e) => e.stopPropagation()}>
```

Comment:
```suggestion
            <ModalContent onClick={(event) => event.stopPropagation()}>
```
We were told by the coaches please not to use these abbreviations.